packages/web/src/components/buy-sell-modal/TokenAmountSection.tsx

```diff
@@ -285,17 +321,22 @@ const CryptoAmountSection = ({
   verticalLayout?: boolean
 }) => {
   const { spacing } = useTheme()
-  const { icon: TokenIcon, symbol } = tokenInfo
+  const { symbol } = tokenInfo
   const tokenTicker = messages.tokenTicker(symbol, !!isStablecoin)
 
-  if (!TokenIcon) {
+  if (!tokenInfo.icon && !tokenInfo.logoURI) {
     return null
   }
 
   if (verticalLayout) {
     return (
       <Flex p={noPadding ? undefined : 'l'} alignItems='center' gap='s'>
-        <TokenIcon width={spacing.unit16} height={spacing.unit16} hex />
+        <TokenIcon
+          tokenInfo={tokenInfo}
+          width={spacing.unit16}
```

Comment:
ideally this could just be w='unit16' w='unit16' using the base props from box? i think icons have that support too